source/_components/sensor.fastdotcom.markdown

```diff
@@ -1,73 +0,0 @@
----
-layout: page
-title: "Fast.com"
-description: "How to integrate Fast.com within Home Assistant."
-date: 2016-08-10 17:30
-sidebar: true
-comments: false
-sharing: true
-footer: true
-logo: fastdotcom.png
-ha_category: System Monitor
-featured: false
-ha_release: 0.26
```

Comment:
Should we keep the original `ha_release`?

Reply:
I think we should just leave it that way